tests/admin_inlines/tests.py

```diff
@@ -854,6 +853,47 @@ def test_delete_inlines(self):
         self.assertEqual(len(self.selenium.find_elements_by_css_selector(
             'form#profilecollection_form tr.dynamic-profile_set#profile_set-2')), 1)
 
+    def test_form_validation_error(self):
+        """
+        Ensure that the delete buttons are also present in the form after
+        a form validation error.
+        """
+        self.admin_login(username='super', password='secret')
+        self.selenium.get('%s%s' % (self.live_server_url,
```

Comment:
I've started using this style: `self.selenium.get(self.live_server_url + reverse('admin:admin_inlines_holder2_add'))`